tests/src/python/providertestbase.py

```diff
@@ -223,7 +223,13 @@ def runGetFeatureTests(self, provider):
         self.assert_query(provider, 'num_char IN (2, 4, 5)', [2, 4, 5])
 
         # geometry
+        self.assert_query(provider, 'azimuth($geometry,geom_from_wkt( \'Point (-70 70)\')) < pi()', [1, 5])
+        self.assert_query(provider, 'x($geometry) < -70', [1, 5])
+        self.assert_query(provider, 'y($geometry) > 70', [2, 4, 5])
+        self.assert_query(provider, 'disjoint($geometry,geom_from_wkt( \'Polygon ((-72.2 66.1, -65.2 66.1, -65.2 72.0, -72.2 72.0, -72.2 66.1))\'))', [4, 5])
         self.assert_query(provider, 'intersects($geometry,geom_from_wkt( \'Polygon ((-72.2 66.1, -65.2 66.1, -65.2 72.0, -72.2 72.0, -72.2 66.1))\'))', [1, 2])
+        self.assert_query(provider, 'touches($geometry,geom_from_wkt( \'Polygon ((-70.332 66.33, -65.32 66.33, -65.32 78.3, -70.332 78.3, -70.332 66.33))\'))', [1, 4])
```

Comment:
Perhaps creating a simpler geometry using string concatenation could work. Something like "touches(geom_from_wkt('linestring(' || some attribute || ' 0, 1 0)'), ...)" Might help? I've only got phone access at the moment so can't make a more helpful expression sorry!